components/math/multiplicationConnect/Modal.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, Ref, useRef } from "react";
+
+interface ModalProps {
+  type: String;
```

Comment:
instead of a string type you can do "centeted" | "fullscreen" | "alert"

Reply:
fixed!